docs/section15/Rule15-1.md

```diff
@@ -0,0 +1,20 @@
+# Transformers - Rule 15-1
+**Rule ID:** 15-1  
+**Rule Description:** Number of transformers modeled in User RMR and Baseline RMR are the same  
+**Rule Assertion:** Baseline RMR = User RMR  
+**Appendix G Section:** Transformers  
+**Appendix G Section Reference:** None  
+**Data Lookup:** None  
+**Evaluation Context:**  Data Group  
+**Applicability Checks:** 
+1. Number of transformers in User RMR greater than 0  
```

Comment:
The purpose of this text block is to provide context for non-developers, so that they can connect how the code represents narrative language within the textual ruleset.  If it is not relevant to a developer then it can be ignored.